resources/scripts/jquery/jquery-utils.js

```diff
@@ -308,6 +308,30 @@
     	$(".eXist_wizard_next", container).click(function () {
     		if (currentPage == pages.length - 1)
     			return;
+		var name = document.getElementsByName('name')[0].value;
+           	var abbrev = document.getElementsByName('abbrev')[0].value;
+           	var title = document.getElementsByName('title')[0].value;
+
+                if (name.startsWith('urn:') || name.startsWith('file:')) {
+                    alert("The name cannot be in a urn:// or a file:// protocol.")
+                    return false;} else {
+                       try {
+                           nameURL = new URL(name);
```

Comment:
`nameURL` is an explicit global variable and is never read. Please remove it. Just `new URL(name);` works, too.

Reply:
Good point; done.